defaults/main.yml

```diff
@@ -1,4 +1,9 @@
 ---
+memcached_user: memcache
```

Comment:
This shouldn't be defined in `defaults/`, but rather `vars/` (which it is, currently), in order to allow RHEL and Debian-specific settings.